frontend/src/assets/css/margin-variables.css

```diff
@@ -0,0 +1,7 @@
+:root {
+    --m-025: 0.25rem;
+    --m-05: 0.5rem;
+    --m-1: 1rem;
+    --m-15: 1.5rem;
+    --m-2: 2rem;
```

Comment:
```suggestion
    --m-1: 0.25rem;
    --m-2: 0.5rem;
    --m-3: 1rem;
    --m-4: 1.5rem;
    --m-5: 2rem;
```

Reply:
Done in commit [d7dfd60](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/d7dfd60823e771fb8617e2e4c36d6cf2f6dff46a)